<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_17) on Sun Apr 19 11:09:46 PDT 2015 -->
<title>org.Spectrums</title>
<meta name="date" content="2015-04-19">
<link rel="stylesheet" type="text/css" href="../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="org.Spectrums";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../overview-summary.html">Overview</a></li>
<li class="navBarCell1Rev">Package</li>
<li>Class</li>
<li><a href="package-use.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../index-files/index-1.html">Index</a></li>
<li><a href="../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../MSPLIT/SpectrumIO/package-summary.html">Prev Package</a></li>
<li><a href="../../UI/package-summary.html">Next Package</a></li>
</ul>
<ul class="navList">
<li><a href="../../index.html?org/Spectrums/package-summary.html" target="_top">Frames</a></li>
<li><a href="package-summary.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h1 title="Package" class="title">Package&nbsp;org.Spectrums</h1>
</div>
<div class="contentContainer">
<ul class="blockList">
<li class="blockList">
<table class="packageSummary" border="0" cellpadding="3" cellspacing="0" summary="Interface Summary table, listing interfaces, and an explanation">
<caption><span>Interface Summary</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Interface</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LibraryPeptideClassifier.PeptideClass.html" title="interface in org.Spectrums">LibraryPeptideClassifier.PeptideClass</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PeakComparator.html" title="interface in org.Spectrums">PeakComparator</a></td>
<td class="colLast">
<div class="block">A general comparator of two peaks, note that in order to
 account for noise model in the scoring, this interface should implements compare
 method that accept null as argument and return an appropriate score accordingly</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumComparator.html" title="interface in org.Spectrums">SpectrumComparator</a></td>
<td class="colLast">
<div class="block">This define a general interface that compare a pair
 of spectrum</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumQualityFilter.html" title="interface in org.Spectrums">SpectrumQualityFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/TableElementOperator.html" title="interface in org.Spectrums">TableElementOperator</a></td>
<td class="colLast">
<div class="block">operator for a individual element of lookup table</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/WiffSDKAccess.WiffReader.html" title="interface in org.Spectrums">WiffSDKAccess.WiffReader</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<table class="packageSummary" border="0" cellpadding="3" cellspacing="0" summary="Class Summary table, listing classes, and an explanation">
<caption><span>Class Summary</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Class</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/AAFreq.html" title="class in org.Spectrums">AAFreq</a></td>
<td class="colLast">
<div class="block">Compute a.a.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/AccurateMassPRM.html" title="class in org.Spectrums">AccurateMassPRM</a></td>
<td class="colLast">
<div class="block">PRM spectrum supporting accurate mass tolerance</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/AnnotatedSpectrum.html" title="class in org.Spectrums">AnnotatedSpectrum</a></td>
<td class="colLast">
<div class="block">\
 A spectrum allowing for various annotation 
 The annotation are stored as a general map of the annotation name and their values</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/ArrayListFactory.html" title="class in org.Spectrums">ArrayListFactory</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/ArraySpectrum.html" title="class in org.Spectrums">ArraySpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/CandidateSpectrumGenerator.html" title="class in org.Spectrums">CandidateSpectrumGenerator</a></td>
<td class="colLast">
<div class="block">Index spectrum library by precursor mass and store them in file
 only load part of the library into memory, as spectral library grows
 it become unfeasible to load them all into memory</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/CandidateSpectrumLibFactory.html" title="class in org.Spectrums">CandidateSpectrumLibFactory</a></td>
<td class="colLast">
<div class="block">Contain method that construct a candidate spectrumlibrary
 peptides are store with simple indexing by their parent mass</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/CandidateSpectrumLibFactoryX.html" title="class in org.Spectrums">CandidateSpectrumLibFactoryX</a></td>
<td class="colLast">
<div class="block">Generate candidates like CandidateSpectrumLibFactory,
 use both parent mass and peaks as filter, the reason the x
 near the class name.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/CombinatoryPeptides.html" title="class in org.Spectrums">CombinatoryPeptides</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/ConsensusSpectrumReader.html" title="class in org.Spectrums">ConsensusSpectrumReader</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/Copy_2_of_MixturePeakScoreLearner.html" title="class in org.Spectrums">Copy_2_of_MixturePeakScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/CopyOfMixturePeakScoreLearner.html" title="class in org.Spectrums">CopyOfMixturePeakScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/CopyOfSpectrum.html" title="class in org.Spectrums">CopyOfSpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/CopyOfTheoreticalSpectrum.html" title="class in org.Spectrums">CopyOfTheoreticalSpectrum</a></td>
<td class="colLast">
<div class="block">Theoretical Spectrum of a peptide</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/CrossLinker.html" title="class in org.Spectrums">CrossLinker</a></td>
<td class="colLast">
<div class="block">A cross-linker is a chemical agent that covalentlly link two amino acid
 residues that are spatially close together</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/DatabaseIndexer.html" title="class in org.Spectrums">DatabaseIndexer</a></td>
<td class="colLast">
<div class="block">A simple database indexer for protein sequences, index all peptides</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/DeconvolutedSpectrum.html" title="class in org.Spectrums">DeconvolutedSpectrum</a></td>
<td class="colLast">
<div class="block">Create decovuluted spectrum</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/DecoySpectrumGenerator.html" title="class in org.Spectrums">DecoySpectrumGenerator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/ExperimentalMixtureAnalysis.html" title="class in org.Spectrums">ExperimentalMixtureAnalysis</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/FeatureXMLParser.html" title="class in org.Spectrums">FeatureXMLParser</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/FreqOffsetGenerator.html" title="class in org.Spectrums">FreqOffsetGenerator</a></td>
<td class="colLast">
<div class="block">Generate frequencey offset statistics for a set of annotated spectrum</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/HarklorParser.html" title="class in org.Spectrums">HarklorParser</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/HashSpectrum.html" title="class in org.Spectrums">HashSpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/HomologMapper.html" title="class in org.Spectrums">HomologMapper</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/IDs2StructureMapper.html" title="class in org.Spectrums">IDs2StructureMapper</a></td>
<td class="colLast">
<div class="block">We map IDs to a structure to use strucural information
 to validate our IDs (particularly crosslinked IDs)</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/InternalLinkedPeptide.html" title="class in org.Spectrums">InternalLinkedPeptide</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/IonType.html" title="class in org.Spectrums">IonType</a></td>
<td class="colLast">
<div class="block">represent a ion type in mass spect fragmentation</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/jianutil.html" title="class in org.Spectrums">jianutil</a></td>
<td class="colLast">
<div class="block">contain various utilitiy function</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LabelledPeak.html" title="class in org.Spectrums">LabelledPeak</a></td>
<td class="colLast">
<div class="block">A peak with annotation</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LabelledPeakFactory.html" title="class in org.Spectrums">LabelledPeakFactory</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LargeHashMap.html" title="class in org.Spectrums">LargeHashMap</a></td>
<td class="colLast">
<div class="block">Large hashmap implementation back-by a random access file</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LargeSpectrumLibIterator.html" title="class in org.Spectrums">LargeSpectrumLibIterator</a>&lt;T&gt;</td>
<td class="colLast">
<div class="block">When spectrum lib is large we avoid loading all of them into memory at once
 instead we read from the files</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LazyEvaluatedSpectrum.html" title="class in org.Spectrums">LazyEvaluatedSpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LazyEvaluateLinkedSpectrum.html" title="class in org.Spectrums">LazyEvaluateLinkedSpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LibraryPeptideClassifier.html" title="class in org.Spectrums">LibraryPeptideClassifier</a></td>
<td class="colLast">
<div class="block">Classify peptides according to definition from NIST 
 The classes are used to help improve quality of spectral library</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedCandidateSpectrumLibFactory.html" title="class in org.Spectrums">LinkedCandidateSpectrumLibFactory</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedCandidateSpectrumLibX.html" title="class in org.Spectrums">LinkedCandidateSpectrumLibX</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedCandidateSpectrumLibX2.html" title="class in org.Spectrums">LinkedCandidateSpectrumLibX2</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedPeakScoreLearner.html" title="class in org.Spectrums">LinkedPeakScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedPepAnalysis.html" title="class in org.Spectrums">LinkedPepAnalysis</a></td>
<td class="colLast">
<div class="block">Containers of some methods that analyze linked peptide</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedPeptide.html" title="class in org.Spectrums">LinkedPeptide</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedPeptideAnalysis.html" title="class in org.Spectrums">LinkedPeptideAnalysis</a></td>
<td class="colLast">
<div class="block">Some utility function for analyzing linked peptides and their annotation</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedPeptidePeakScoreLearner.html" title="class in org.Spectrums">LinkedPeptidePeakScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedRankBaseScoreLearner.html" title="class in org.Spectrums">LinkedRankBaseScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedSpectrumIonRankLearner.html" title="class in org.Spectrums">LinkedSpectrumIonRankLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LinkedTheoreticalSpectrum.html" title="class in org.Spectrums">LinkedTheoreticalSpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LMixturePeakScoreLearner.html" title="class in org.Spectrums">LMixturePeakScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LookUpPeak.html" title="class in org.Spectrums">LookUpPeak</a></td>
<td class="colLast">
<div class="block">Compute look up peaks of a give spectrum, use to filter
 out candidates</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LookUpSpectrumLib.html" title="class in org.Spectrums">LookUpSpectrumLib</a></td>
<td class="colLast">
<div class="block">A spectrum lib, indexed by peaks in the spectrum rather than
 peptides</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LookUpSpectrumLibX.html" title="class in org.Spectrums">LookUpSpectrumLibX</a></td>
<td class="colLast">
<div class="block">A spectrum lib, indexed by peaks in the spectrum rather than
 peptides</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LookUpSpectrumLibXX.html" title="class in org.Spectrums">LookUpSpectrumLibXX</a></td>
<td class="colLast">
<div class="block">Algorithm base on paper by Nathan Edward and Ross  Lippert LNCS 2452, pp.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LookUpTable.html" title="class in org.Spectrums">LookUpTable</a></td>
<td class="colLast">
<div class="block">General N-dimensional table, backed by an one dimensional array
 access to the table through index manipulation</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LookUpTableUtils.html" title="class in org.Spectrums">LookUpTableUtils</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/LPeakRankBaseScorer.html" title="class in org.Spectrums">LPeakRankBaseScorer</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/LPeakScoreLearner.html" title="class in org.Spectrums">LPeakScoreLearner</a></td>
<td class="colLast">
<div class="block">A wrapper class that score linked peptides base on 
 single-peptide models</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/Mass.html" title="class in org.Spectrums">Mass</a></td>
<td class="colLast">
<div class="block">mass information for amino acid and various type of ions</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MemoryUsageTester.html" title="class in org.Spectrums">MemoryUsageTester</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MGF2DTA.html" title="class in org.Spectrums">MGF2DTA</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MixDB.html" title="class in org.Spectrums">MixDB</a></td>
<td class="colLast">
<div class="block">main interface to MixDB</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MixSSM.html" title="class in org.Spectrums">MixSSM</a></td>
<td class="colLast">
<div class="block">Describe a mixture or multiplexed spectrum-spectrum matches</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MixturePeak.html" title="class in org.Spectrums">MixturePeak</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MixturePeakScoreLearner.html" title="class in org.Spectrums">MixturePeakScoreLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MixturePeptideAnalysis.html" title="class in org.Spectrums">MixturePeptideAnalysis</a></td>
<td class="colLast">
<div class="block">Contain various method that analyze mixtrure spectrum</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MixtureSpectrum.html" title="class in org.Spectrums">MixtureSpectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MixtureSpectrumComparator.html" title="class in org.Spectrums">MixtureSpectrumComparator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MixtureSpectrumScorer.html" title="class in org.Spectrums">MixtureSpectrumScorer</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MixtureSVMClassify.html" title="class in org.Spectrums">MixtureSVMClassify</a></td>
<td class="colLast">
<div class="block">Perform SVM classification for mixture spectrum
 using svm_light</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MixtureTDA.html" title="class in org.Spectrums">MixtureTDA</a></td>
<td class="colLast">
<div class="block">Perform SVM classification of</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/ModifiedSpectrum.html" title="class in org.Spectrums">ModifiedSpectrum</a></td>
<td class="colLast">
<div class="block">This class help create modified version of a spectrum by
 allowing for a mass shift in the peptide</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MS2Summary.html" title="class in org.Spectrums">MS2Summary</a></td>
<td class="colLast">
<div class="block">print out some summary for mzxml file</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MSFeature.html" title="class in org.Spectrums">MSFeature</a></td>
<td class="colLast">
<div class="block">Peptidic features in LC/MS maps</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MSFeatureAcq.html" title="class in org.Spectrums">MSFeatureAcq</a></td>
<td class="colLast">
<div class="block">A MSFeature with acquisition information</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MSFeatureAnalyzer.html" title="class in org.Spectrums">MSFeatureAnalyzer</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MSGFInputGenerator.html" title="class in org.Spectrums">MSGFInputGenerator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MSLabMain.html" title="class in org.Spectrums">MSLabMain</a></td>
<td class="colLast">
<div class="block">This class contain usage info for using the various function in the whole 
 MsLab package</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MSP2Mgf.html" title="class in org.Spectrums">MSP2Mgf</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MSPLITHybrid.html" title="class in org.Spectrums">MSPLITHybrid</a></td>
<td class="colLast">
<div class="block">We test a hybrid method that utilize both features from spectral matches and 
 DBsearch to discriminate true matches from false ones</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MSScanInfo.html" title="class in org.Spectrums">MSScanInfo</a></td>
<td class="colLast">
<div class="block">Add MS scan level information to IDs</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MultiSWATHAnalyzer.html" title="class in org.Spectrums">MultiSWATHAnalyzer</a></td>
<td class="colLast">
<div class="block">This class contain method to analyze multiple SWATH/DIA runs</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MXDBPairSearch.html" title="class in org.Spectrums">MXDBPairSearch</a></td>
<td class="colLast">
<div class="block">Search for mixture and crosslinked peptides using pair peptide as input</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MXDBSearch.html" title="class in org.Spectrums">MXDBSearch</a></td>
<td class="colLast">
<div class="block">Search for mixture and crosslinked peptide</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MXDBSearchSingle.html" title="class in org.Spectrums">MXDBSearchSingle</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MXDBSumo.html" title="class in org.Spectrums">MXDBSumo</a></td>
<td class="colLast">
<div class="block">Perform MXDBSearch for SUMOylated peptides</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MXDBSumo2.html" title="class in org.Spectrums">MXDBSumo2</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MXGF.html" title="class in org.Spectrums">MXGF</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MXGF_sumo.html" title="class in org.Spectrums">MXGF_sumo</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MXGFe.html" title="class in org.Spectrums">MXGFe</a></td>
<td class="colLast">
<div class="block">An extension to MXGF with edge scores that score pair of consecutive
 aa along a peptides, can handle hi-mass accuracy data</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/MZMLReader.html" title="class in org.Spectrums">MZMLReader</a></td>
<td class="colLast">
<div class="block">MZML reader for mass spectrum</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/MZXML2MGF.html" title="class in org.Spectrums">MZXML2MGF</a></td>
<td class="colLast">
<div class="block">Convert mzxml file to mgf</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/Pair.html" title="class in org.Spectrums">Pair</a></td>
<td class="colLast">
<div class="block">Generaic class for storing a pair of object</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PairedPeptide.html" title="class in org.Spectrums">PairedPeptide</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/Peak.html" title="class in org.Spectrums">Peak</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PeakComparatorSet.html" title="class in org.Spectrums">PeakComparatorSet</a></td>
<td class="colLast">
<div class="block">A wrapper that contain multiple PeakComparator, it
 can dispatch the appropriate comparator base-on the peptide
 length and charge the particular peak belong to</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/PeakIntensityComparator.html" title="class in org.Spectrums">PeakIntensityComparator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PeakMassComparator.html" title="class in org.Spectrums">PeakMassComparator</a></td>
<td class="colLast">
<div class="block">Compare peak in a spectrum by its mass</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/PeakRankBaseComparator.html" title="class in org.Spectrums">PeakRankBaseComparator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PeakScoreLearner.html" title="class in org.Spectrums">PeakScoreLearner</a></td>
<td class="colLast">
<div class="block">Similar to SpectrumIonsLearner, learn how to score a matching peaks
 back by a generic lookup table</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/Peptide.html" title="class in org.Spectrums">Peptide</a></td>
<td class="colLast">
<div class="block">Represent a peptide</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PeptideLite.html" title="class in org.Spectrums">PeptideLite</a></td>
<td class="colLast">
<div class="block">A simple memory efficient implementation of peptides</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/PeptideMassAnalysis.html" title="class in org.Spectrums">PeptideMassAnalysis</a></td>
<td class="colLast">
<div class="block">This class provide general methods for analyze peptides</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PeptideMassComparator.html" title="class in org.Spectrums">PeptideMassComparator</a></td>
<td class="colLast">
<div class="block">Impose ordering of peptides according to their masses
 note that in order to allow the notion that different peptides
 can has exactly the same masses, when mass is equal the comparator do 
 not return zero, which is reserve for cases when the two peptides are the same</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/PreciseCandidatesFactory.html" title="class in org.Spectrums">PreciseCandidatesFactory</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PrecursorMassChecker.html" title="class in org.Spectrums">PrecursorMassChecker</a></td>
<td class="colLast">
<div class="block">Check precursor mass for MS/MS spectrum annotation
 in the case of high-accuracy MS scan we can use this
 as a first-level of validation of IDs returned by our search algorithm
 Alternatively we can also use  this as a preprocessing step to select select
 candidate for scoring, since precurosr recorded in MS/MS maybe not be 
 corrected or the first isotopic mass of peptide being fragment we
 wil scan the MS1 profiel for matches</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/PRMSpectrum.html" title="class in org.Spectrums">PRMSpectrum</a></td>
<td class="colLast">
<div class="block">Compute PRM Spectrum</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/PRMSpectrumComparator.html" title="class in org.Spectrums">PRMSpectrumComparator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/ProteinDigester.html" title="class in org.Spectrums">ProteinDigester</a></td>
<td class="colLast">
<div class="block">Digest a proteins into peptides</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/ProteinIDExtractor.html" title="class in org.Spectrums">ProteinIDExtractor</a></td>
<td class="colLast">
<div class="block">Given a sequence database and a list of search results from
 database search engine:
 - Extract list of protein IDs from a list of PSMs
 - Mapped the peptide IDs to protein positions
 * @author Jian Wang</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/PTM.html" title="class in org.Spectrums">PTM</a></td>
<td class="colLast">
<div class="block">A object represent a specific PTM on a peptide</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/RankBaseMixtureSpectrumScorer.html" title="class in org.Spectrums">RankBaseMixtureSpectrumScorer</a></td>
<td class="colLast">
<div class="block">Rank base scorer for mixture spectrum, due to the rank-base
 probability model is base on single peptide, ranks in mixture
 spectrum might not be corrected, need to adjust the peak rank
 base on the two peptides being considered</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/RankBaseScoreLearner.html" title="class in org.Spectrums">RankBaseScoreLearner</a></td>
<td class="colLast">
<div class="block">Similar to SpectrumIonRankLearner but reimplemented using
 Lookup table</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/ResultColumnIndex.html" title="class in org.Spectrums">ResultColumnIndex</a></td>
<td class="colLast">
<div class="block">A class search result column index used to parse result file</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/RTAligner.html" title="class in org.Spectrums">RTAligner</a></td>
<td class="colLast">
<div class="block">This class align two set of RT between observed data and 
 library spectra.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SearchResult.html" title="class in org.Spectrums">SearchResult</a></td>
<td class="colLast">
<div class="block">A class represent search result of a MS spectrum file</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SearchResultReporter.html" title="class in org.Spectrums">SearchResultReporter</a></td>
<td class="colLast">
<div class="block">Report search results related stats</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SimpleMatchGraphFactory.html" title="class in org.Spectrums">SimpleMatchGraphFactory</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SimpleMatchingGraph.html" title="class in org.Spectrums">SimpleMatchingGraph</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SimplePeakComparator.html" title="class in org.Spectrums">SimplePeakComparator</a></td>
<td class="colLast">
<div class="block">A simple peak comparator that score a given theoretical peak against a experimental peaks.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SimpleProbabilisticScorer.html" title="class in org.Spectrums">SimpleProbabilisticScorer</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SimpleProbabilisticScorer2.html" title="class in org.Spectrums">SimpleProbabilisticScorer2</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SortedMZXMLReader.html" title="class in org.Spectrums">SortedMZXMLReader</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpecPlotAnnotation.html" title="class in org.Spectrums">SpecPlotAnnotation</a></td>
<td class="colLast">
<div class="block">Generate annotation file for specplot</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/Spectrum.html" title="class in org.Spectrums">Spectrum</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumCosineComparator.html" title="class in org.Spectrums">SpectrumCosineComparator</a></td>
<td class="colLast">
<div class="block">Cosine comparator</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumFDRFilter.html" title="class in org.Spectrums">SpectrumFDRFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumIonRankLearner.html" title="class in org.Spectrums">SpectrumIonRankLearner</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumIonsLearner.html" title="class in org.Spectrums">SpectrumIonsLearner</a></td>
<td class="colLast">
<div class="block">Analyze annotated spectrum, extract ions type information or statistics</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLib.html" title="class in org.Spectrums">SpectrumLib</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibConstructor.html" title="class in org.Spectrums">SpectrumLibConstructor</a></td>
<td class="colLast">
<div class="block">Construct a spectral library from a set of PSMs
 with RT alignment to a reference run</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibMap.html" title="class in org.Spectrums">SpectrumLibMap</a></td>
<td class="colLast">
<div class="block">create a map for spectrumlib so spectrum in the lib
 can be lookup using some property like scan number</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibQCFilter.html" title="class in org.Spectrums">SpectrumLibQCFilter</a></td>
<td class="colLast">
<div class="block">A set of filter designed to ensure the quality of the spectral library
 This default set of filters are adapted from those used in NIST
 User can add or remove filters from the default sets</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibQCFilter.PepIonSignificanceFilter.html" title="class in org.Spectrums">SpectrumLibQCFilter.PepIonSignificanceFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibQCFilter.PrecursorMassErrorFilter.html" title="class in org.Spectrums">SpectrumLibQCFilter.PrecursorMassErrorFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibQCFilter.PrincipleChargeStateFilter.html" title="class in org.Spectrums">SpectrumLibQCFilter.PrincipleChargeStateFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibQCFilter.SufficientIonsFilter.html" title="class in org.Spectrums">SpectrumLibQCFilter.SufficientIonsFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibQCFilter.UnIdentFragmentIonsFilter.html" title="class in org.Spectrums">SpectrumLibQCFilter.UnIdentFragmentIonsFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumLibSearcher.html" title="class in org.Spectrums">SpectrumLibSearcher</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumMap.html" title="class in org.Spectrums">SpectrumMap</a></td>
<td class="colLast">
<div class="block">An sortedmap back implementation of spectrum that enalbe quick lookup
 of peak by mass or intensity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumMassComparator.html" title="class in org.Spectrums">SpectrumMassComparator</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumPepFilter.html" title="class in org.Spectrums">SpectrumPepFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumProteinFilter.html" title="class in org.Spectrums">SpectrumProteinFilter</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SpectrumUtil.html" title="class in org.Spectrums">SpectrumUtil</a></td>
<td class="colLast">
<div class="block">Various utility method for spectrum</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SSMExtractor.html" title="class in org.Spectrums">SSMExtractor</a></td>
<td class="colLast">
<div class="block">Extract the matching peaks in a Spectrum-Spectrum Matches
 from a spectral library search result and</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SWATHMGFGenerator.html" title="class in org.Spectrums">SWATHMGFGenerator</a></td>
<td class="colLast">
<div class="block">Take in a SWATH file and a output file from Harklor
 http://proteome.gs.washington.edu/software/hardklor/
 Generate a IDA-like mgf file that associate all precursor
 in the SWATH window with each SWATH MS/MS spectrum</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SWATHMSPLITSearch.html" title="class in org.Spectrums">SWATHMSPLITSearch</a></td>
<td class="colLast">
<div class="block">Initial try on searching SWATH data with M-SPLIT</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SWATHPreprocessor.html" title="class in org.Spectrums">SWATHPreprocessor</a></td>
<td class="colLast">
<div class="block">Do some preprocessing for SWATH spectrum
 it seems the centroided SWATH spectrum is not very good
 some information may be lost in its native peak-picking form</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/SWATHStatistics.html" title="class in org.Spectrums">SWATHStatistics</a></td>
<td class="colLast">
<div class="block">This class holds a series of utility static method to compute various statistics for 
 SWATH spectra</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/SWATHUtils.html" title="class in org.Spectrums">SWATHUtils</a></td>
<td class="colLast">
<div class="block">Some utility function for SWATH data</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/TableElementOperator.Divider.html" title="class in org.Spectrums">TableElementOperator.Divider</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/TableElementOperator.Log.html" title="class in org.Spectrums">TableElementOperator.Log</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/TDAStat.html" title="class in org.Spectrums">TDAStat</a></td>
<td class="colLast">
<div class="block">Estimate FDR using TDA, various info related to FDR is computed</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/TestSpectrumLib.html" title="class in org.Spectrums">TestSpectrumLib</a></td>
<td class="colLast">
<div class="block">Various method for testing SpectrumLib</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/testSpectrumLibSearch.html" title="class in org.Spectrums">testSpectrumLibSearch</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/TheoreticalCandidateSpectrumFactory.html" title="class in org.Spectrums">TheoreticalCandidateSpectrumFactory</a></td>
<td class="colLast">
<div class="block">Generate theoretical spectrum candidates;
 back by a Peptide candidates table stored in file</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/TheoreticalFragmentSpectrum.html" title="class in org.Spectrums">TheoreticalFragmentSpectrum</a></td>
<td class="colLast">
<div class="block">Generate theoretical spectrum for fragment ions rather than normal ions</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/TheoreticalSpectrum.html" title="class in org.Spectrums">TheoreticalSpectrum</a></td>
<td class="colLast">
<div class="block">Theoretical Spectrum of a peptide</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../org/Spectrums/ToMgf.html" title="class in org.Spectrums">ToMgf</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../org/Spectrums/WiffSDKAccess.html" title="class in org.Spectrums">WiffSDKAccess</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../overview-summary.html">Overview</a></li>
<li class="navBarCell1Rev">Package</li>
<li>Class</li>
<li><a href="package-use.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../index-files/index-1.html">Index</a></li>
<li><a href="../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../MSPLIT/SpectrumIO/package-summary.html">Prev Package</a></li>
<li><a href="../../UI/package-summary.html">Next Package</a></li>
</ul>
<ul class="navList">
<li><a href="../../index.html?org/Spectrums/package-summary.html" target="_top">Frames</a></li>
<li><a href="package-summary.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
